spec/requests/create_collection_spec.rb

```diff
@@ -108,34 +115,39 @@
           before { collection_params.merge!(review_workflow_params) }
 
           it 'nils out the reviewers field' do
-            post '/collections', params: { collection: collection_params, commit: 'Deposit' }
+            post '/collections', params: { collection: collection_params, commit: deposit_button }
             expect(response).to have_http_status(:found)
             expect(response).to redirect_to(dashboard_path)
             collection = Collection.last
             expect(collection.reviewers).to be_empty
           end
         end
 
-        context 'with empty fields' do
+        context 'with mostly empty fields' do
           let(:draft_collection_params) do
             {
               name: '',
               description: '',
               contact_email: '',
               manager_sunets: user.sunetid,
               access: 'world',
-              depositor_sunets: ''
+              depositor_sunets: 'maya.aguirre,jcairns, cchavez, premad, giancarlo, zhengyi'
```

Comment:
This does not belong in the "mostly empty fields". This was a test to show it was okay to provide nothing, and now we don't have it.

Reply:
ok, i'll revert this test and add a new context block with a new test